searx/static/themes/simple/gruntfile.js

```diff
@@ -117,7 +117,7 @@ module.exports = function (grunt) {
           outputSourceFiles: true,
         },
         files: {
-          "css/searxng.min.css": "src/less/style.less",
+          "css/searxng.min.css": "src/less/style-ltr.less",
```

Comment:
I miss the `src/less/style-ltr.less` file .. or do I misunderstand this?

Reply:
Face palm, I forgot to add this file. Fixed.